src/utils/errorsFormatter.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Stringify the Error
+ * @param error
+ * @returns A formatted string of the error.
+ */
+function getMessage(error: Error): string {
+  return `value [${JSON.stringify(error)}]`;
+}
+
+/**
+ * Merge into one single Error several errors provided in input and add a context description
+ * @param errors
+ * @param context
+ * @returns A single Error instance with a formatted message.
+ */
+export function multipleErrorsFormatter(
+  errors: ReadonlyArray<Error>,
+  context: string
+): Error {
+  return new Error(
+    errors
+      .map(_ => {
```

Comment:
since `getMessage` is called only from here and is so small it can be inlined

```suggestion
      .map(_ => `value [${JSON.stringify(_)}]`)
```

Reply:
fixed in bda2514